### ISSUE TITLE
Search box not auto focused.

### ISSUE THREAD

**Issue by kxxoling** [storybooks/storybook#3486]
### Bug or support request summary

Search box not auto focused on open. I think the shortcut is aimed to make actions easy, but not auto focusing makes things inconvenient.

### Steps to reproduce

Just using `cmd+shift+o`:
![image](https://user-images.githubusercontent.com/1227139/39225299-14662abe-487e-11e8-9e1f-87be176db9b9.png)

this is manually focused: 
![image](https://user-images.githubusercontent.com/1227139/39225306-1fa48632-487e-11e8-8733-918b6df8b02d.png)

### Packages installed

```
    "@babel/core": "^7.0.0-beta.46",
    "@babel/preset-react": "^7.0.0-beta.46",
    "@storybook/addon-actions": "^4.0.0-alpha.3",
    "@storybook/addon-links": "^4.0.0-alpha.3",
    "@storybook/addons": "^4.0.0-alpha.3",
    "@storybook/react": "^4.0.0-alpha.3",
```

**Comment by Keraito** [comment]
I can take a look at this

**Comment by Hypnosphi** [comment]
Released as [`4.0.0-alpha.4`](https://github.com/storybooks/storybook/releases/tag/v4.0.0-alpha.4)
